From fb14098c71153e8668867a12cb66c0da70ba0817 Mon Sep 17 00:00:00 2001
From: Vihang Karajgaonkar <vihang@cloudera.com>
Date: Mon, 11 Jul 2016 14:26:20 -0500
Subject: [PATCH 0717/1164] CDH-42419 : HIVE-14151: Use of USE_DEPRECATED_CLI
 environment variable does not work (Vihang
 Karajgaonkar, reviewed by Sergio Pena)

Change-Id: I9281d06fcc0cd5aa7c6b6d917fa911072978a8b1
---
 bin/ext/cli.sh |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bin/ext/cli.sh b/bin/ext/cli.sh
index 353f297..ef3870c 100644
--- a/bin/ext/cli.sh
+++ b/bin/ext/cli.sh
@@ -17,7 +17,8 @@ THISSERVICE=cli
 export SERVICE_LIST="${SERVICE_LIST}${THISSERVICE} "
 
 # Set old CLI as the default client
-if [ -n '$USE_DEPRECATED_CLI' ]; then
+# if USE_DEPRECATED_CLI is not set or is not equal to false use old CLI
+if [ -z "$USE_DEPRECATED_CLI" ] || [ "$USE_DEPRECATED_CLI" != "false" ]; then
   USE_DEPRECATED_CLI="true"
 fi
 
-- 
1.7.9.5

